js/background.js

```diff
@@ -143,6 +143,21 @@
             SERVER_URL, SERVER_PORTS, USERNAME, PASSWORD
         );
 
+        // Because v0.43.2 introduced a bug that lost contact details, v0.43.4 introduces
+        //   a one-time contact sync to restore all lost contact/group information. We
+        //   disable this checking if a user is first registering.
+        var key = 'chrome-contact-sync-v0.43.4';
+        if (firstRun) {
+            storage.put(key, true);
```

Comment:
This will re-write 'true' every time we re-register. Slightly better for perf if we drop this and add a `&& !firstRun` to L#156.

Also, I'm not sure if this was the intent but, I note that:
1. this code runs every time we reconnect to the internet after being offline for a while.
2. firstRun will be true on a re-registration, not just a first-registration.